src/Repository/Criteria.php

```diff
@@ -266,4 +297,20 @@ public function getSince(): \DateTimeImmutable
             default => throw new \LogicException(),
         };
     }
+
+    public function getOption(string $key): string
+    {
+        return match ($key) {
+            'sort' => $this->resolveSort($this->sortOption),
+            'time' => '∞' === $this->resolveTime($this->time, true) ? 'all' : $this->resolveTime($this->time, true),
```

Comment:
why is `reverse` here `true`?

Reply:
See above talking about conversion of '∞', this is using that functionality

Reply:
please add a comment here why this is necessary. Right now it is just borderline unreadable :sweat_smile:

Reply:
Hm, how's this sound?

```
    // resolveTime() converts our internal values into ones for human presenation
    // $reverse = true indicates converting back, from human values to internal ones

    // This whole approach is a mess; this translation layer is temporary until
    // we have time to take a pass through the whole codebase and convert so there's
    // no such thing as multiple alternate value strings and translation layers
    // between them. This is just a temporary measure to produce desired output
    // until the whole layer goes away.
    public function resolveTime(?string $value, bool $reverse = false): ?string

```

Also, you'll be entertained to learn that Copilot auto-suggested this comment when I typed "// So:":

```
    // So: This whole method is a mess. It's not clear what it's supposed to do, and it's not clear what the input and output are supposed to be.
```

So it agrees with you 🙂

Reply:
Added

Reply:
Thank you copilot for feeling bad

Reply:
I mean... tell you what. One other option for how to do the whole thing would be to refactor *first* -- consolidate down to just one representation / one string value for each filtering option everywhere, and *then* do the filter changes with this whole ugly translation layer already removed.

Basically make things better and then add the feature, as opposed to making things worse with adding the feature, with the vague plan at some point in the distant future of making it back better again. I don't like the idea of blowing up the scope of this PR until everything it touches can be beautiful again, but if this way of doing the translation is too complex and weird, to the point that we want to wait on a separate change to clean things up first, I can kind of understand that.

I was biased in favor of trying to get the filtering changes working + merged, but the other way works too; it just might take a little longer. Just LMK.